### PR TITLE
feat; Missing migrations for anonymous-consents #9841

### LINKED ISSUE #9841: Missing migrations for anonymous-consents
Changes not covered with migrations/docs:
image

Missing information about dropping `storageSync` (for example search `storageSync` in 3_0.md):
image

### PR DESCRIPTION
**Pull request by Pucek9**
Closes #9841

### CODE REVIEW

**Review by kacperknapik — DISMISSED** [review]

**Comment by kacperknapik** [comment]
Wait for the decision about merge destination - we think about merging all migration tickets to epic branch first.

File: docs/migration/3_0.md [hunk 1]
```diff
@@ -179,6 +179,7 @@ There can be more than one cart entry with the same product code. So now they ar
 - `AuthConfig` no longer extends `OccConfig`.
 - `login` and `revoke` endpoints were removed from `OccConfig`. `login` endpoint is now available under `tokenEndpoint` property in `AuthConfig`. `revoke` endpoint is available under `revokeEndpoint` property in `AuthConfig`.
 - `storageSync` configuration for `auth` branch in ngrx store was removed. State of token, userId is now synchronized with `AuthStatePersistenceService`. Override this service if you want to sync more properties to localStorage (eg. `refresh_token`).
+- `anonymousConsentsStoreConfigFactory` was removed. Function has been replaced by `anonymousConsentsStatePersistenceFactory`. It requires param type `AnonymousConsentsStatePersistenceService` and it doesn't return `StateConfig`, only function allows for initialization the synchronization between state and browser storage.
```
Comment (marlass) [inline docs/migration/3_0.md]: These details are not important. More important is mention of storageSync config. Eg.
```suggestion
- `storageSync` configuration for `anonymous-consents` branch in ngrx store was removed. State is now synchronized with `AnonymousConsentsStatePersistenceService`. Override this service if you want to sync more/less properties to localStorage.
```

**Review by marlass — CHANGES_REQUESTED** [review]
Also add information about adapter/connector change in the migration doc (3_0.md)

**Comment by Matejk00** [comment]
Please check requested changes.

**Review by marlass — DISMISSED** [review]

**Review by Matejk00 — APPROVED** [review]

**Review by marlass — APPROVED** [review]

**Pull request merged**